uni/lib/view/common_widgets/pages_layouts/general/widgets/top_navigation_bar.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+
+/// Upper bar of the app.
+///
+/// This widget consists on an instance of `AppBar` containing the app's logo,
+/// an option button and a button with the user's picture.
+class AppTopNavbar extends StatelessWidget implements PreferredSizeWidget {
+  const AppTopNavbar({
+    this.title,
+    this.rightButton,
+    this.leftButton,
+    super.key,
+  });
+
+  static const double borderMargin = 18;
+
+  final String? title;
+  final Widget? rightButton;
+  final Widget? leftButton;
+
+  @override
+  Size get preferredSize => const Size.fromHeight(kToolbarHeight);
+
+  @override
+  AppBar build(BuildContext context) {
+    return AppBar(
+      automaticallyImplyLeading: false,
+      elevation: 0,
+      iconTheme: Theme.of(context).iconTheme,
+      backgroundColor: Theme.of(context).scaffoldBackgroundColor,
+      titleSpacing: 0,
+      title: Stack(
+        alignment: Alignment.center,
+        children: [
+          Align(
+            alignment: Alignment.centerLeft,
+            child: leftButton,
+          ),
+          Center(
+            child: Text(
+              title ?? '',
+              textAlign: TextAlign.center,
+              style: Theme.of(context).textTheme.headlineMedium?.copyWith(
+                    color: Theme.of(context)
+                        .primaryTextTheme
+                        .headlineMedium
+                        ?.color,
+                  ),
+            ),
+          ),
```

Comment:
I appreciate that this is very well generalized here, but I am concerned about `PageTitle` still being needed in some cases. I wonder if it is best to use `PageTitle` here or try to delete it altogether and fit its remaining usage in this schema.

Reply:
I would say delete `PageTitle`, as it seems it's not going to be used in the new design, the title in the top bar seems to replace it. Should I delete it in this PR?

Reply:
I've tried to delete `PageTitle` and found out that it is used also in the page subtitles and such. So, I'll try the route to use `PageTitle` in the top navbar.